src/main/java/ro/societateahermes/backendservice/entities/Timeline.java

```diff
@@ -0,0 +1,21 @@
+package ro.societateahermes.backendservice.entities;
+
+import lombok.Data;
+import ro.societateahermes.backendservice.entities.dto.EventDTO;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import java.util.List;
+
+@Data
+@Entity
+public class Timeline {
+    @Id
+    @GeneratedValue
+    private Long IdTimeline;
+    @OneToMany(mappedBy = "timeline")
+    private List<EventDTO> listOfEvents;
+}
```

Comment:
Timeline-ul apartine de un singur event si este reprezentat de lista de activitati. Nu asa ar trebui sa arate modelul. Ar trebui sa modifici logica functionalitatii ca sa iti obtina timelineul in functie de id-ul evenimentului, nu al unui obiect Timeline